piecedirectory/doctor.go

```diff
@@ -75,7 +109,8 @@ func (d *Doctor) Run(ctx context.Context) {
 		// Sleep for a few seconds between ticks.
 		// The time to sleep is randomized, so that if there are multiple doctor
 		// processes they will each process some pieces some of the time.
-		sleepTime := avgCheckInterval/2 + time.Duration(rand.Intn(int(avgCheckInterval)))*time.Millisecond
```

Comment:
This is a bug.

Reply:
Nice catch 🎣 

I think we still want the randomization behaviour - it's for the case where there are multiple boost processes all running piece doctor. The bug is the multiplication by milliseconds.